test/data/data.js

```diff
@@ -29,6 +29,15 @@ function populate(db) {
     }),
   );
 
+  prom.push(
+    insertDocument(db, {
+      $type: 'entry',
+      $owners: ['a@a.com'],
+      $id: 'documentOfA',
+      $content: {},
+    }),
+  );
```

Comment:
you need to specify the `_id` corresponding to the token's `uuid` to make the test work correctly